api/v1alpha2/istio_merge.go

```diff
@@ -2,13 +2,14 @@ package v1alpha2
 
 import (
 	"encoding/json"
+
 	"google.golang.org/protobuf/types/known/wrapperspb"
 	"istio.io/api/operator/v1alpha1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 
 	"google.golang.org/protobuf/types/known/structpb"
 	meshv1alpha1 "istio.io/api/mesh/v1alpha1"
-	istioOperator "istio.io/istio/operator/pkg/apis/istio/v1alpha1"
+	iopv1alpha1 "istio.io/istio/operator/pkg/apis/istio/v1alpha1"
```

Comment:
Yes, this is correct. I did it just because previously it was confusing using camil-case import, e.g. `istioOperator.something`. Was looking like accessing struct instance. I did rename import with the version because of consistency, but I see reasoning not to use version in the name so we do not have to rename all if we change the version. Perhaps I could refactor all imports not to use version, what do you think?

Reply:
We should avoid changing import without aligning on import. But perhaps refactor all api versioned imports in another issue/task/PR.

Reply:
Yes, I think we can talk about it in the team and then just change it in one PR. I'll track it.